js/everything.js

```diff
@@ -137,6 +136,7 @@ var ZenPlayer = {
                 that.setupTitle();
                 that.setupVideoDescription(videoID);
                 that.setupPlyrToggle();
+                that.show();
```

Comment:
Showing the player defeats the purpose of this project, can we instead start playing here? I had that working once upon a time, then reverted it once we set `autoplay=true`

Reply:
we can start playing here by calling `play`, but still we'll need the user to enable the `autoplay` on the site already to do that, we'll get `Autoplay is only allowed when approved by the user, the site is activated by the user, or media is muted.`  

one way we can do it is, we can `play` here, but in the `play` function we'll need to check if autoplay is not allowed by user, then we show the player to let user click the play button.